### PR TITLE
Tng 973 update dependencies

### PR DESCRIPTION
**Pull request by rrieger-espirit**
This PR updates the deprecated dependencies.. Husky also moved to another version, so there had some optimizations to be done

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,41 +38,43 @@
...
-    "fsxa-api": "^5.1.0",
-    "fsxa-pattern-library": "^3.1.0",
+    "fsxa-api": "^5.3.0",
+    "fsxa-pattern-library": "^3.2.2",
+    "glob-parent": "^6.0.0",
     "lodash.camelcase": "^4.3.0",
     "lodash.merge": "^4.6.2",
     "lodash.startcase": "^4.4.0",
-    "vuex": "^3.5.1"
+    "prismjs": "^1.24.1",
```
Comment (dbarczewski) [inline package.json:76]: Why are here new dependencies with prismjs and glob-parent?

**Comment by dbarczewski** [comment]
Please remove the fsxa-nuxt-module-3.5.0.tgz file, as it should not be pushed. 
Maybe even edit the `.gitignore` file so that `.tgz` cannot be pushed.

**Comment by dbarczewski** [comment]
Since there will be a new version of the fsxa-pattern-library and fsxa-api it makes more sense to publish them first and increase the version here.

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```
Comment (henczi-espirit) [inline .husky/pre-commit:2]: what does this preCommit script do?

File: package.json [hunk 3]
```diff
@@ -37,41 +38,43 @@
...
+    "@semantic-release/github": "^7.2.3",
+    "@semantic-release/npm": "^7.1.3",
+    "@semantic-release/release-notes-generator": "^9.0.3",
+    "@types/jest": "^26.0.23",
     "@types/lodash.merge": "^4.6.6",
-    "@types/node": "^14.0.27",
-    "@typescript-eslint/eslint-plugin": "^3.9.1",
-    "@typescript-eslint/parser": "^3.9.1",
-    "commitizen": "^4.1.2",
+    "@types/node": "^16.0.1",
```
Comment (henczi-espirit) [inline package.json:52]: Deployment target of most fsxa-pwa is node 14. Did you test, does the node module still work for node 14? Is there an automated test for it?

**Comment by dbarczewski** [comment]
Dependencies were update in #8 . I will close this PR.

**Pull request closed without merging**